packages/fxa-settings/src/models/Account.ts

```diff
@@ -1192,6 +1192,47 @@ export class Account implements AccountData {
     return recoveryKey;
   }
 
+  // TODO FXA-7546/FXA-7669 - Update methods to only emit a "recovery key changed" email
```

Comment:
Oh yeah, we'll definitely want confirmation ASAP on this. I think we'll want a ticket for a new email template (2-3 points) and a ticket for setting up a GQL mutation for us to hit + send the email and update the client-side call (~3 points).

Not great here that we're sending 4 different requests...
![094346E3-BCE6-4CF1-B839-8A0B471DE9AE](https://github.com/mozilla/fxa/assets/13018240/b973ea42-4c55-48bf-a5ee-ede38987e8bd)

This leaves us with an edge case too of if the DELETE call succeeds and the new POST call fails. Maybe this is okay for now but hopefully we can just send up `keyFetchToken` and `unwrapBKey` from `reauth` with the session token and let everything else happen server-side.

Reply:
I've moved some of the logic server-side, we're back down to 3 requests instead of 4.

Changing a key now only sends out a "new key created" email (not "key deleted"), which is aligned with the CTA when changing the key. We might still want those tickets though to serve a more specific email that clarifies that the previous key is now invalid.